### PR TITLE
Add seed servers term

### PR DESCRIPTION
**Pull request by JakeSCahill**

### CODE REVIEW

File: modules/terms/partials/advertised-listener.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
 === advertised listener
 :term-name: advertised listener
-:hover-text: A publicly accessible endpoint that brokers advertise to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API.
+:hover-text: An endpoint that brokers advertise to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API.
```
Comment (JakeSCahill) [inline modules/terms/partials/advertised-listener.adoc]:
```suggestion
:hover-text: The address a Redpanda broker broadcasts to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API. The advertised address might differ from the listener address in scenarios where brokers are behind a NAT, in a Docker container, or Kubernetes. Advertised addresses ensure clients can reach the Redpanda brokers even in complex network setups.
```

**Review by micheleRP — APPROVED** [review]
thanks, Jake! I left one little suggestion--not sure if it's worth making

**Comment by JakeSCahill** [comment]
Thanks @micheleRP - I can’t see the suggestion. Is it still there?

File: modules/terms/partials/advertised-listener.adoc [hunk 2]
```diff
@@ -1,4 +1,4 @@
 === advertised listener
 :term-name: advertised listener
-:hover-text: A publicly accessible endpoint that brokers advertise to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API.
+:hover-text: The address a Redpanda broker broadcasts to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API. The advertised address might differ from the listener address in scenarios where brokers are behind a NAT, in a Docker container, or Kubernetes. Advertised addresses ensure clients can reach the Redpanda brokers even in complex network setups.
```
Comment (micheleRP) [inline modules/terms/partials/advertised-listener.adoc]:
```suggestion
:hover-text: The address a Redpanda broker broadcasts to producers, consumers, and other brokers. It specifies the hostname and port for connections to different listeners. Clients and other brokers use advertised listeners to connect to services such as the Admin API, Kafka API, and HTTP Proxy API. The advertised address might differ from the listener address in scenarios where brokers are behind a NAT, in a Docker container, or in Kubernetes. Advertised addresses ensure clients can reach the Redpanda brokers even in complex network setups.
```

**Comment by micheleRP** [comment]
> Thanks @micheleRP - I can’t see the suggestion. Is it still there?

weird, sorry. I must not have saved it! I just added that one tiny thing now!

**Pull request merged**